### DISCUSSION TITLE
Custom regex example in ~/.config/tailspin/config.toml

### DISCUSSION THREAD

**Discussion by rikosintie** [bensadeh/tailspin#189] [Q&A] [answered]
Hello, thanks for creating this project!

I am using it with logs from Cisco IOS network switches.

I would like to color entries like "GigabitEthernet1/0/25"

The pattern can have 1 digit after GigabitEthernet, 1 digit after the first / and two digits at the end. For example
GigabitEthernet8/0/48
GigabitEthernet1/1/1
GigabitEthernet2/1/4

I created IOS-config.toml and placed it in ~/.config/tailspin. 

Here is the entry for the regex

[[regexes]]
regex = 'GigabitEthernet\d{1,2}/\d{1,2}/\d{1,2}'
style = { fg = "red" }

It works in sublime text but not when I run tspin. 

Other entries in the toml file do work.
[[keywords]]
words = ['user', 'User', 'Interface']
style = { fg = "blue", bold = true, italic = false }

Can you point me in the right direction?

Thanks!

**Comment by chadouze** [comment, marked as answer]
Hello there !
Can you show a screenshot with output, please ?
I guess, built-in highlight groups overlapping custom regexes, therefore I think you should use --disable flag for them.
